commcare_connect/data_export/views.py

```
@@ -206,6 +215,44 @@ def get_queryset(self, request, opp_id):
+
+    def get_queryset(self, request, opp_id):
+        filters = {}
+        query_params = request.query_params.copy()
+        query_params.pop("opportunity", None)
+        for key, value in query_params.items():
+            filters[key] = value
+        return LabsRecord.objects.filter(opportunity=self.opportunity, **filters).annotate(
+            username=F("user__username"),
+        )
+
+    def create(self, request, *args, **kwargs):
```

Ruff (0.14.4)

Unused method argument: `args`

(ARG002)